server.js

```diff
@@ -18,6 +18,10 @@ if (fs.existsSync(config)) {
     args = JSON.parse(fs.readFileSync(config));
 }
 
+if (!process.env['DISPLAY']){
```

Comment:
Does this work on Windows?

Reply:
It runs inside docker env. So it doesn't matter the host OS, the OS that will run will be the one mentioned in `Dockerfile`.
In this case, it will be a lightweight linux distro

Reply:
> It runs inside docker env.

But others will run `server.js` outside of docker.

Reply:
Good point. I misread the code.

Reply:
They might use Windows and utilize `TamperDAV.bat` to start the server. In this case `dialog` should be used.

Reply:
I'll look at swapping it out with a docker env variable.

Reply:
Actually, I think checking for PID 1 would work better.  Seems Docker doesn't currently have an official way to check if your running inside a container.

Reply:
Nah, that's a crude heuristic. Instead you can add a `--headless` config option.

Reply:
That also works.

Reply:
Should be updated now.